libs/@types/jsfeat/package.json

```diff
@@ -1,5 +1,6 @@
 {
     "name": "@types/jsfeat",
```

Comment:
@eventualbuddha does this package need to exist? I don't see it imported anywhere.

Reply:
It does not. We stopped using `jsfeat` when we switched the ballot interpreter to Rust.